stko/optimizers/xtb.py

```diff
@@ -516,6 +518,422 @@ def optimize(self, mol):
         return mol
 
 
+class XTBCREST(Optimizer):
+    """
+    Uses GFN-n [1]_ to run CREST [2]_ on molecules.
+
+    Notes
+    -----
+    Requires version > 6.2 of xtb.
+
+    When running :meth:`optimize`, this calculator changes the
+    present working directory with :func:`os.chdir`. The original
+    working directory will be restored even if an error is raised, so
+    unless multi-threading is being used this implementation detail
+    should not matter.
+
+    If multi-threading is being used an error could occur if two
```

Comment:
Cool, I think it should be removed to not confuse the user. I don't believe multi-threading is used within stko so far and this could confuse users.